samples/Monitoring/deploy-mmaextension-windows-tags-loganalytics/azurepolicy.json

```diff
@@ -0,0 +1,140 @@
+{
+    "properties": {
+        "displayName": "Deploy Microsoft Monitoring Agent on Windows Machines based on a Tag",
```

Comment:
> Microsoft Monitoring Agent [](start = 30, length = 27)

please change all references to MMA to "Log Analytics Agent"

Reply:
The correct terminology according to the Log Analytics team is "Log Analytics Agent".  It is what they use in their built-in policies and documentation.

---
In reply to: [310304741](https://github.com/Azure/azure-policy/pull/435#discussion_r310304741) [](ancestors = 310304741)

Reply:
I've corrected this - Agent name to be Log Analytics Agent.